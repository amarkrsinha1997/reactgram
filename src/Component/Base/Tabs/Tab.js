import React from 'react';
import { colors } from '../../../styles/colors';

const style =  `
	.tab {
		background: ${colors.grey.shade1};
		padding: 5px 10px;
		color: ${colors.black.shade1};
	}
`
const Tab = (props) => {
	return (
		<div className="tab">
			{props.children}
			<style>{style}</style>
		</div>
	)
}

export { Tab }

