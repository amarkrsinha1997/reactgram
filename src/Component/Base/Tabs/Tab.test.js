import React from 'react';
import { render } from '@testing-library/react';
import { Tab } from './Tab';

test('renders learn react link', () => {
    const { container } = render(<Tab />);
    expect(container).toMatchSnapshot();
});
