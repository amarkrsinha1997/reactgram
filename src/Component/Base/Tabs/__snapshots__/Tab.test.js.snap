// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div>
  <div
    class="tab"
  >
    <style>
      
	.tab {
		background: #dedede;
		padding: 5px 10px;
		color: #333333;
	}

    </style>
  </div>
</div>
`;
